### PR TITLE
Update the Windows build section

### PR DESCRIPTION
**Pull request by vinnik-dmitry07**
Structured building steps, fixed arguments in cmake command, used environment variables to follow the DRY principle, added an output example, fixed link to the DEVELOPING.md.

### CODE REVIEW

**Review by vigsterkr — CHANGES_REQUESTED** [review]
great! i'm just not so sure about including the whole cmake listing... this is really a lot of LoC :)
but while you are at it, do you think you could add couple of lines about clcache? we are using it in azure pipelines so from the yaml files you can figure out how it is used. it is very handy when you need to compile shogun couple of times. it's like ccache but for windows.

**Comment by vinnik-dmitry07** [comment]
> great! i'm just not so sure about including the whole cmake listing... this is really a lot of LoC :)
> but while you are at it, do you think you could add couple of lines about clcache? we are using it in azure pipelines so from the yaml files you can figure out how it is used. it is very handy when you need to compile shogun couple of times. it's like ccache but for windows.

Should I merely add some mention about it and put this nice [link](https://github.com/frerich/clcache/wiki/Caveats#no-cache-hits-when-building-via-visual-studio-ide-or-msbuild)?

**Comment by vigsterkr** [comment]
linking it would be more than sufficient thanx!

**Comment by vigsterkr** [comment]
@vinnik-dmitry07 any update on this? would be great to merge, just the clcache part is missing :)

**Comment by vinnik-dmitry07** [comment]
@vigsterkr Yes, I have almost completed.

**Comment by vinnik-dmitry07** [comment]
That is it.

File: doc/readme/INSTALL.md [hunk 1]
```diff
@@ -286,27 +286,64 @@ On a Linux cluster without root access, using [Anaconda](https://www.continuum.i
...
+3. Find the path of the _Native Tools Command Prompt for VS_ relative to your system and Visual studio (x32/x64, 2017/2019). For example for _x64 Native Tools Command Prompt for VS 2019_ it looks so: `%comspec% /k "X:\Path\To\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvars64.bat"`
 
+4. Execute this path in the _Anaconda Prompt_ to run _Native Tools_ there 
 
-Compiling works as
+5. Put these lines into the _Anaconda Prompt_:
+> rem - Records comments (remarks); @rem - do not print command just it's result (the rem result is empty)
 
-    msbuild "C:\projects\shogun\build\shogun.sln" /verbosity:minimal /t:Clean /p:Configuration=Release /p:Platform=x64
+> Be carefull cmake can search out of the build directory to load libraries thus remove other directories that contain the shogun sources
```
Comment (vigsterkr) [inline doc/readme/INSTALL.md]: careful

**Review by vigsterkr — COMMENTED** [review]
one little typo

**Pull request merged**